curta/src/chip/hash/sha/mod.rs

```diff
@@ -1 +1,2 @@
 pub mod sha256;
+pub mod keccak256;
```

Comment:
Small comment but it would be better to have Keccak under `hash` (one level up) rather than under `sha`